app/deepseg.cc

```diff
@@ -448,7 +514,7 @@ int main(int argc, char* argv[]) try {
 
 	cv::Mat mask(height, width, CV_8U);
 	cv::Mat raw;
-	CalcMask ai(modelname, threads, width, height);
+	CalcMask ai(s_model.c_str(), threads, width, height);
```

Comment:
Sure that we can't avoid passing raw pointers around?

Reply:
Agreed, we ought to stick with `const std::string &` unless it's an external API that uses `const char *`, will fix for background loader too.